_includes/meetup2017009.html

```diff
@@ -53,7 +53,7 @@
         {% include components/speakerimage.html imagename="m009_filipalacerda.jpg" %}
         <div class="media-content">
           <div class="content">
-            <h3>Culture: a gentle introduction</h3>
```

Comment:
damn file copy.. would be nice in the future to think about how we can use jekyl to avoid repeating titles and similar content

Reply:
You mean reuse components/resources?